internal/command/run.go

```diff
@@ -157,6 +162,23 @@ func run(cmd *cobra.Command, args []string) error {
 
 	// Validate SCORE spec
 	//
+	if !skipValidation {
+		log.Print("Validating SCORE spec...\n")
+		if res, err := schema.Validate(gojsonschema.NewGoLoader(srcMap)); err != nil {
+			return fmt.Errorf("validating workload spec: %w", err)
+		} else if !res.Valid() {
+			for _, valErr := range res.Errors() {
+				log.Println(valErr.String())
+				if err == nil {
```

Comment:
Is it intentional to spread only the first error returned by the validation (and log all of them)?

Reply:
Correct. All errors are shown in logs (in verbose mode), and we return any of those as an error to terminate the CLI. We could return an abstract error, something like "validation failed", but additional details are more helpful. And packing all the errors into one is not an option here, as there could be too many errors.